lib/i18n-spec/matchers/be_a_complete_translation_of_matcher.rb

```diff
@@ -1,14 +1,32 @@
-RSpec::Matchers.define :be_a_complete_translation_of do |default_locale_filepath|
+RSpec::Matchers.define :be_a_complete_translation_of do |default_locale_filepath, check_interpolation_key = false|
```

Comment:
Using keyword argument seems **to me** fine tho it's, in general, difficult problem to take Ruby version up to least 2.x unnecessarily.
And about the default value, I just want this feature to fail existing specs.